docs/themes/thxvscode/layouts/partials/break-on-capitals.html

```diff
@@ -0,0 +1,19 @@
+{{ $input := . }}
```

Comment:
Can we add some top-level documentation explaining what this is doing? I think such documentation is especially important for code like these Hugo templates, since most of the team is not familiar with them.

Reply:
Thanks for adding the docs! They look great!